.github/pull_request_template.md

```diff
@@ -1,3 +1,11 @@
+## Cover letter
+
+Describe in plain language:
+
+1. The problem that the feature addresses
+2. How the feature addresses the problem
+3. How to use the feature
+
 ## Checklist
 - [ ] Reference related [issue](https://github.com/vectorizedio/redpanda/issues)
 - [ ] Update [PendingReleaseNotes.md](https://github.com/dotnwat/redpanda/blob/dev/PendingReleaseNotes.md), if relevant
```

Comment:
I'd like to take this opportunity to propose changing from the current checklist to having:

```suggestion
**Issue(s)**: [LIST OF RELEVANT ISSUES HERE]
```

Reply:
i like this.